src/main/target/FOXEERF745_AIO/target.h

```diff
@@ -0,0 +1,161 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+ #pragma once
+
+ #define TARGET_BOARD_IDENTIFIER "FOXE"
+ #define USBD_PRODUCT_STRING "FOXEERF745_AIO"
+
+ #define LED0_PIN   PC13
+
+ #define USE_BEEPER
+ #define BEEPER_PIN PD2
+ #define BEEPER_INVERTED
+
+ #define CAMERA_CONTROL_PIN PB3
+
+ #define MPU6000_CS_PIN        PA15
+ #define MPU6000_SPI_INSTANCE  SPI3
+
+ #define USE_ACC
+ #define USE_ACC_SPI_MPU6000
+ #define ACC_MPU6000_ALIGN CW180_DEG
+
+ #define USE_GYRO
+ #define USE_GYRO_SPI_MPU6000
+ #define GYRO_MPU6000_ALIGN CW180_DEG
+
+ // MPU6000 interrupts
+ #define USE_MPU_DATA_READY_SIGNAL
+ #define MPU_INT_EXTI PD0
+ #define USE_EXTI
+
+ #define USE_MAG
+ #define USE_MAG_HMC5883
+ #define USE_MAG_QMC5883
+ #define USE_MAG_LIS3MDL
+ #define MAG_I2C_INSTANCE           (I2CDEV_1)
+ //#define MAG_HMC5883_ALIGN CW270_DEG_FLIP
+ //#define MAG_ALIGN CW180_DEG  //not sure if this command will work or if should be more specific to mag
```

Comment:
nope. just inspected code today.

we have these on these lines
```
src/main/sensors/compass.c:70:    compassConfig->mag_align = ALIGN_DEFAULT;
src/main/sensors/compass.c:154:    dev->magAlign = ALIGN_DEFAULT;
src/main/sensors/compass.c:164:#ifdef MAG_HMC5883_ALIGN
src/main/sensors/compass.c:165:            dev->magAlign = MAG_HMC5883_ALIGN;
src/main/sensors/compass.c:178:#ifdef MAG_QMC5883L_ALIGN
src/main/sensors/compass.c:179:            dev->magAlign = MAG_QMC5883L_ALIGN;
src/main/sensors/compass.c:192:#ifdef MAG_AK8975_ALIGN
src/main/sensors/compass.c:193:            dev->magAlign = MAG_AK8975_ALIGN;
src/main/sensors/compass.c:211:#ifdef MAG_AK8963_ALIGN
src/main/sensors/compass.c:212:            dev->magAlign = MAG_AK8963_ALIGN;
src/main/sensors/compass.c:251:    if (compassConfig()->mag_align != ALIGN_DEFAULT) {
src/main/sensors/compass.c:252:        magDev.magAlign = compassConfig()->mag_align;
src/main/sensors/compass.c:271:    alignSensors(mag.magADC, magDev.magAlign);
```
no generic `MAG_ALIGN`.